examples/express/.env.example

```
@@ -0,0 +1,6 @@
+SCALEKIT_ENVIRONMENT_URL=https://your-env.scalekit.cloud
+SCALEKIT_CLIENT_ID=skc_xxx
+SCALEKIT_CLIENT_SECRET=your_client_secret
+REDIRECT_URI=http://localhost:5001/callback
+COOKIE_ENCRYPTION_SECRET=generate_with_openssl_rand_base64_32
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The COOKIE_ENCRYPTION_SECRET key should go before the REDIRECT_URI key

(UnorderedKey)